packages/sitecore-jss-nextjs/src/middleware/multisite-middleware.test.ts

```diff
@@ -509,6 +522,44 @@ describe('MultisiteMiddleware', () => {
         pathname: '/_site_foo/styleguide',
       });
     });
+
+    it('should assign attributes to site cookie', async () => {
+      const req = createRequest({
+        cookieValues: { sc_site: 'foobar' },
+      });
```

Comment:
we should add test case for:
1) when `siteCookieAttributes` are set to undefined.
2) when the expiration time is past date.
3) Test for boolean attributes (if they are false then they should be present)

Reply:
makes sense. but since we'll make this non configurable and just set attributes by default I think I can even remove this test and just adjust all the others